<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSession</class>
 <widget class="QWidget" name="AddSession">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый сеанс</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/images/ico.png</normaloff>:/images/ico.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="minimumDate">
      <date>
       <year>2023</year>
       <month>12</month>
       <day>1</day>
      </date>
     </property>
     <property name="firstDayOfWeek">
      <enum>Qt::Monday</enum>
     </property>
     <property name="gridVisible">
      <bool>true</bool>
     </property>
     <property name="dateEditAcceptDelay">
      <number>1500</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Время</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTimeEdit" name="film_timeEdit">
         <property name="maximumTime">
          <time>
           <hour>18</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="minimumTime">
          <time>
           <hour>8</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Зал</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="hall_num_spinBox"/>
       </item>
       <item row="2" column="1">
        <widget class="QTimeEdit" name="duration_timeEdit">
         <property name="maximumTime">
          <time>
           <hour>5</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Продолжительность</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="confirm_pushButton">
     <property name="text">
      <string>Добавить сеанс</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
